config/jest/jest.config.js

```diff
@@ -18,9 +19,10 @@ process.env.PUBLIC_URL = '';
 process.env.BROWSERSLIST = 'current node';
 
 const pkg = packageRoot();
+const iLibDirs = ['node_modules/@enact/i18n/ilib', 'node_modules/ilib', 'ilib'];
 const globals = {
 	__DEV__: true,
-	ILIB_BASE_PATH: 'node_modules/@enact/i18n/ilib',
+	ILIB_BASE_PATH: iLibDirs.find(f => fs.existsSync(path.join(pkg.path, f))) || iLibDirs[0],
```

Comment:
Can you explain why you chose to use the old location as the fallback in case none exist?

Reply:
Whoops, that's a remnant of the change earlier today. Fallback should be tthe new location (and when a fallback, primarily use for global value usage. I'll update